package.json

```
@@ -16,7 +16,7 @@
     "prebuild": "npm run clean",
     "build": "NODE_ENV=production webpack --optimize-minimize",
     "build-dev": "NODE_ENV=development webpack --progress",
-    "pretest": "npm run lint",
+    "apretest": "npm run lint",
```

```suggestion
    "pretest": "npm run lint",
```
good for dev, not for commit ;-)